src/main/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJob.java

```diff
@@ -53,65 +45,23 @@
 
 @Configuration
 @EnableBatchProcessing
-@Import({VepAnnotationGeneratorStep.class, VepInputGeneratorStep.class, AnnotationLoaderStep.class})
-public class AnnotationJob extends CommonJobStepInitialization{
+@Import({AnnotationFlow.class})
+public class AnnotationJob extends CommonJobStepInitialization {
     public static final String jobName = "annotate-variants";
-    public static final String SKIP_ANNOT = "annotation.skip";
-    public static final String GENERATE_VEP_ANNOTATION = "Generate VEP annotation";
-    private static final String OPTIONAL_VARIANT_VEP_ANNOTATION_FLOW = "Optional variant VEP annotation flow";
-    private static final String VARIANT_VEP_ANNOTATION_FLOW = "Variant VEP annotation flow";
 
     @Autowired
     private JobBuilderFactory jobBuilderFactory;
 
-    @Qualifier("vepInputGeneratorStep")
     @Autowired
-    public Step variantsAnnotGenerateInputBatchStep;
-
-    @Qualifier("annotationLoad")
-    @Autowired
-    private Step annotationLoadBatchStep;
-
-    @Autowired
-    private VepAnnotationGeneratorStep vepAnnotationGeneratorStep;
-
+    private Flow optionalAnnotationFlow;
+    
     @Bean
     public Job variantAnnotationBatchJob(){
         JobBuilder jobBuilder = jobBuilderFactory
                 .get(jobName)
                 .incrementer(new RunIdIncrementer());
 
-        return jobBuilder.start(optionalAnnotationFlow()).build().build();
-    }
-
-    @Bean
-    public Flow annotationFlow(){
```

Comment:
Oops, I wrote this in the wrong place, the right one is https://github.com/EBIvariation/eva-pipeline/pull/29/files#diff-21cf8aa01a49da282421272bac7718b7R43